Project-II-Calculator/calculator/src/App.scss

```diff
@@ -0,0 +1,29 @@
+body{
+    //background-color: red;
+
+    width:241px;
+    height:316px;
+    margin: 30px auto;
+    border: 1px solid lightgray;
+}
+
+button{
+    height:50px;
+    font-size: 25px;
+    border:.5px solid lightgrey;
```

Comment:
Considering that this `button` element selector is for your `ActionButton`/`NumberButton` component, you might want to place it inside the `Button.scss` file in your `ButtonComponents` directory instead.